plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/my-subscriptions.scss

```diff
@@ -258,3 +292,34 @@
 		padding: 0;
 	}
 }
+
+
+.woocommerce-marketplace__my-subscriptions {
+	display: grid;
+	grid-template-columns: repeat(5, auto);
+	.woocommerce-marketplace__my-subscriptions-section {
+		margin: 1em 0;
+		> * {
+			grid-column: span 5;
+		}
+	}
+	.woocommerce-table__table,
+	.woocommerce-marketplace__my-subscriptions-section {
+		&,
+		& table,
+		& tbody,
+		& thead,
+		& tr {
+			grid-column: span 5;
+			display: grid;
+			grid-template-columns: subgrid;
+			max-width: calc(100vw - (2 * #{$grid-unit-20}));
+			@media screen and (min-width: 783px) {
+				max-width: calc(100vw - (2 * #{$grid-unit-40}) - $admin-menu-width-collapsed);
+			}
+			@media screen and (min-width: 960px) {
+				max-width: calc(100vw - (2 * #{$grid-unit-40}) - $admin-menu-width);
+			}
+		}
+	}
```

Comment:
Wow, this turned into a whole thing, didn't it? Sorry it had to go so "deep"!

Reply:
It wasn't an easy problem to solve, but we got there in the end.
Thanks for figuring out the subgrid part!